tests/test_tablepyxl.py

```diff
@@ -161,23 +161,25 @@ def test_style_string_to_dict(self):
 
     def test_style_dict_to_style(self):
```

Comment:
Should we add a test where we call style_dict_to_named_style with the same attributes and assert that we get something already in known_styles instead of a new style?